app/assets/javascripts/short-form/ShortFormApplicationController.js.coffee

```diff
@@ -219,6 +228,54 @@ ShortFormApplicationController = (
     else if $scope.application.answeredCommunityScreening ==  'Yes'
       ShortFormNavigationService.goToApplicationPage('dahlia.short-form-welcome.overview')
 
+  ########## BEGIN CUSTOM EDUCATOR SCREENING LOGIC ##########
+
+  # reserved for educators
+  $scope.customEducatorIsListing1 = ->
+    $scope.listing.Custom_Listing_Type == 'Educator 1: SFUSD employees only'
```

Comment:
Should we reference a constant here instead? I see these strings are in `constants.tsx`

Reply:
We can't reference values defined in typescript files from inside coffeescript files.

I did consider creating constants in the Angular part of the repo, but decided not to because I wanted to minimize Angular code changes for this 'bandaid' feature.